src/it/scala/nl/wbaa/testkit/AwaitAtMostTrait.scala

```diff
@@ -0,0 +1,18 @@
+package nl.wbaa.testkit
+
+import scala.concurrent.duration._
+import scala.util.Try
+
+
+trait AwaitAtMostTrait {
+  private val maxWaitTimeForContainers: Int = 10 * 60
```

Comment:
This should be:
 - on an object,
 - `private[this]`,
 - a `FiniteDuration`.
 - named _default_ instead of _max_.